CRUDRecipeEF.BL.DL/Entities/Menu.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CRUDRecipeEF.BL.DL.Entities
+{
+    public class Menu
+    {
+        public int Id { get; set; }
+        public List<Recipe> Recipes { get; set; } = new List<Recipe>();
```

Comment:
We decided to add a name property right?

Reply:
where do you want from me to add it?

Reply:
Here in the menu since the restaurant has multiple menus we need to identify each menu somehow to the user

Reply:
i added a name with required and pushed it